common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -164,7 +163,15 @@ public ManagedBuffer getBlockData(String appId, String execId, String blockId) {
     int shuffleId = Integer.parseInt(blockIdParts[1]);
     int mapId = Integer.parseInt(blockIdParts[2]);
     int reduceId = Integer.parseInt(blockIdParts[3]);
+    return getBlockData(appId, execId, shuffleId, mapId, reduceId);
+  }
 
+  /**
+   * Obtains a FileSegmentManagedBuffer from (shuffleId, mapId, reduceId). We make assumptions
+   * about how the hash and sort based shuffles store their data.
+   */
+  public ManagedBuffer getBlockData(String appId, String execId, int shuffleId, int mapId,
```

Comment:
nit: style. See constructor at top of file for the style when param lists are long.